tests/unit/test_query_language.py

```diff
@@ -381,3 +384,36 @@ def test_incompatible_duration_operations(lhs, op, rhs):
         else:
             assert False
         assert not result
+
+
+@pytest.mark.parametrize(
+    "fn_name",
+    {
+        k
+        for k, v in DateFunctions.__dict__.items()
+        if callable(v) and "add_" not in k and "sub_" not in k
```

Comment:
I'd like a comment about why we're ignoring the `add_` and `sub_` methods.

Reply:
Better might be to have a list of method names that we ignore.  (See my comment below about making the reader do less thinking!)